src/Bicep.Decompiler.IntegrationTests/Bicep.Decompiler.IntegrationTests.csproj

```diff
@@ -9,6 +9,17 @@
     <EmbeddedResource Include="./Files/**/*.*" LogicalName="$([System.String]::new('Files/%(RecursiveDir)%(Filename)%(Extension)').Replace('\', '/'))" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Files\Working\naming001.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="Files\Working\naming001-simple\main.bicep">
+      <LogicalName>$([System.String]::new('Files/%(RecursiveDir)%(Filename)%(Extension)').Replace('\', '/'))</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Files\Working\naming001-simple\main.json" />
+  </ItemGroup>
```

Comment:
Was this autogenerated by VS, and are these changes necessary? This csproj should already be embedding all files inside `Files` with a wildcard pattern.

Reply:
Ugh, thanks.  Hate that VS does that, will remove.

Reply:
done